internal/internal.go

```diff
@@ -277,3 +277,9 @@ const RLSLoadBalancingPolicyName = "rls_experimental"
 type EnforceSubConnEmbedding interface {
 	enforceSubConnEmbedding()
 }
+
+// EnforceClientConnEmbedding is used to enforce proper ClientConn implementation
+// embedding.
+type EnforceClientConnEmbedding interface {
+	enforceSubConnEmbedding()
```

Comment:
s/SubConn/ClientConn/

Reply:
Fixing it in https://github.com/grpc/grpc-go/pull/8026.